### DISCUSSION TITLE
traffic light with timer

### DISCUSSION THREAD

**Discussion by Didoul35** [slyh/Cities2-TrafficLightsEnhancement#111] [Ideas]
Hello,

Have you planned, or even started, the traffic light time management functionality?

If so, it could be interesting to keep your current systems but just to have a gauge (as you did for pedestrians) to facilitate the management of traffic light intersections, then then create a traffic light time management system. tricolors with personality settings by the user (like TMPE in CS1).

THANKS!

**Comment by slyh** [comment]
I'm sorry that I have remained silent for a long time. The good news is that I have resumed work on the custom phases feature and plan to begin testing in September.

Regarding timing management, I am currently unsure how much effort a flow based system will require. However, it should be trivial to implement a minimum duration slider similar to that used in pedestrian crossings.

**Reply by Rafario448** [reply]
thats great news! Specially now with the road builder mod. A per lane custom phasing will be wonders to making intersections like the Continuous Flow intersection.
![firefox_WDUjOTb1db](https://github.com/user-attachments/assets/c26e9cb8-7178-4858-bd89-380f4246b5c7)

**Comment by sacefgop** [comment]
@slyh2 Hello, I have seen the TLE 0.3 preview version in the game and would like to suggest if this feature can be implemented: you can copy or save the traffic light clearance rules you have set yourself, and resetting them at each intersection is very troublesome.thank you！
